### ISSUE TITLE
Only show days of the current month

### ISSUE THREAD

**Issue by jcmendes9898** [twinssbc/Ionic2-Calendar#519]
Hi, 
Is there way to show only the days of the month we are seeing? so in the view you also can see the next and previous month dates in order to complete the row .

**Comment by twinssbc** [comment]
@jcmendes9898 Sorry, I don't have a clear picture of your requirement. How to also see the next and previous month dates? Maybe some mockup pages could help.

**Comment by jcmendes9898** [comment]
It's like it presents you 6 rows, normally the last onde shows the days of the next month. I want to show 5 rows instead. I tried to change de number that u had on the component that is 42 on the for.(i assume that is 6 rows mulitplied by the 7 days of the week). i change it to 35 (to be 5 rows multiplied by the 7 days of the week). but it just give me erros.

**Comment by jcmendes9898** [comment]
Forget, i make it work. But if it only shows 5 rows, some months will miss the 30 or 31 day....

**Comment by twinssbc** [comment]
@jcmendes9898 Yes, that's why if you check all calendar components, it normally shows 7 rows.

**Comment by jcmendes9898** [comment]
@twinssbc and if you keep the number of rows and show only the days of the month. Instead of reduce the number of rows. Like if other month, instead of number u pass empty string
